sdk/storage/Azure.Storage.Blobs/src/Models/BlockBlobOpenWriteOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Optional parameters for Page Blob Open Write.
+    /// </summary>
+    public class BlockBlobOpenWriteOptions
+    {
+        /// <summary>
+        /// Whether the upload should overwrite any existing blobs.
+        /// The default value is false.
+        /// </summary>
+        public bool Overwrite { get; set; }
+
+        /// <summary>
+        /// The size of the buffer to use.  Default is 1 MB,
+        /// max is 100 MB.
```

Comment:
reference the block blob client property for this

Reply:
Same as above.